src/components/raspberry-pi/type/body-heading/_index.scss

```diff
@@ -0,0 +1,12 @@
+@use 'src/mixins/margin';
+@use 'src/primitives/heading';
+@use 'src/mixins/type-scale';
+@use 'src/tokens/line-height';
+
+.sc-rp-type-body-heading {
+  @include heading.primitive;
+  font-size: inherit;
```

Comment:
I'd rather we were more explicit about setting the values of things. I've had issues in the past where `inherit` can tie in knots, and `initial` is far too vague. If we can set this to `skateboard` that would be better I reckon!

Reply:
@Jonic happy to do that although Greg's brief was this heading should always match whatever the body is. With the above, we create a dependency between this and the body in terms of keeping them 'in sync', if the body should ever change. If you're happy with that, I'll do it.

Reply:
Yeah we'll handle how we keep `font-size`s in sync when we're introducing the new type and spacing system. We can't really make the changes to the type scaling system without also addressing the type components themselves, so we'll definitely make sure it's all covered there. As it currently stands, the `heading` primitive says nothing about font-size, so the `font-size` will always match that of the `<body>` anyway.

Does this throw any spanners in the works for your JAMstack work?

Reply:
@Jonic all gravy - let's ditch this PR for now.